tests/topotests/lib/bgp.py

```diff
@@ -850,6 +850,55 @@ def verify_as_numbers(tgen, topo, input_dict):
     return True
 
 
+def clear_bgp(tgen, addr_type, router, vrf=None):
+    """
+    This API is to clear bgp neighborship by running
+    clear ip bgp */clear bgp ipv6 * command,
+
+    Parameters
+    ----------
+    * `tgen`: topogen object
+    * `addr_type`: ip type ipv4/ipv6
+    * `router`: device under test
+    * `vrf`: vrf name
+
+    Usage
+    -----
+    clear_bgp(tgen, addr_type, "r1")
+    """
+
+    logger.debug("Entering lib API: clear_bgp()")
+    if router not in tgen.routers():
+        return False
+
+    rnode = tgen.routers()[router]
+
+    if vrf:
+        if type(vrf) is not list:
+            vrf = [vrf]
+
+    # Clearing BGP
+    logger.info("Clearing BGP neighbor-ship for router %s..", router)
+    if addr_type == "ipv4":
+        if vrf:
+            for _vrf in vrf:
+                run_frr_cmd(rnode, "clear ip bgp vrf {} *".format(_vrf))
+        else:
+            run_frr_cmd(rnode, "clear ip bgp *")
+    elif addr_type == "ipv6":
+        if vrf:
+            for _vrf in vrf:
+                run_frr_cmd(rnode, "clear bgp vrf {} ipv6 *".format(_vrf))
+        else:
+            run_frr_cmd(rnode, "clear bgp ipv6 *")
+    else:
+        run_frr_cmd(rnode, "clear bgp *")
+
+    sleep(5)
```

Comment:
What's the point of this sleep?  Is there a better way to approach this?

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.

Reply:
This sleep is removed as this is not required here at this point.